### PR TITLE
Kudos enhancements

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description

The goal of this PR is to address remaining issues throughout the kudos app.

##### TODOs:

- [ ] qa stuff on mainnet (see #product-kudos-qa )
- [x] once QAed, re-enable nav ( revert 4b4e41d5ca5743f2a785e830b61746ed1e4f1be8 fb9ebbc41a4c0f50451ba0f48e7163a65f699404  , and aslo remember to remove the `display: none` in top_nav.html
- [ ] deploy grants logo to top_nav.html, per IRL convos with mark/vivek
- [ ] bulk send, so we can use it in our devcon preso `https://github.com/mbeacom/gitcoin-erc721/issues/209`
- [x] Fix C++ badge
- [x] Fix txid message length on `receive` kudos page
- [ ] General bug fixes found during QA
- [ ] Introduce grants landing

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Affected core subsystem(s)

kudos

##### Testing

- [ ] Local
- [ ] Stage

### CODE REVIEW

**Comment by octavioamu** [comment]
2 and 5 done

File: app/kudos/kudos.yaml [hunk 1]
```diff
@@ -645,7 +645,7 @@
   - memory management
   - common
   - programming
-  image: c++_scientist.svg
+  image: cpp_scientist.svg
```
Comment (owocki) [inline app/kudos/kudos.yaml]: we will need to re-mint?

Comment (mbeacom) [reply app/kudos/kudos.yaml]: Yes - I was planning on minting following the merge.

**Comment by mbeacom** [comment]
@owocki Should we address https://github.com/gitcoinco/web/issues/2558 here?

**Comment by owocki** [comment]
@mbeacom i've addressed https://github.com/gitcoinco/web/issues/2558 in 8726f150a80d5fda61f724213703fd4ccfa981ec

**Pull request merged**